packages/blob/README.md

```diff
@@ -31,11 +31,30 @@ async function put(
     // `token` defaults to process.env.BLOB_READ_WRITE_TOKEN on Vercel
     // and can be configured when you connect more stores to a project
     // or using Vercel Blob outside of Vercel
-    // During browser uploads, a token is automatically generated by calling the route from the `handleBlobUploadUrl` option
+    // You could also pass a client token here if you generated it yourself with `generateClientTokenFromReadWriteToken`
```

Comment:
it seems a bit confusing this point, why don't we allow the client upload only from the `clientPut`? if the user wants to pass the clientToken explicitly, we can create a type like `{handleClientUploadUrl} | {clientToken}`

Reply:
I see `clientPut` as a convenience method companion to the handleBlobUpload (server) method. But nothing prevents someone from using the put() method directly from the frontend. Either via a regular token or a client token. There's a good probability some people might just want to roll their custom client upload doing so, and I don't want to artificially limit them doing so.

Reply:
so it's possible to send a `cacheControlMaxAge` option from the client by manually creating a token. The option would be then ignored by the worker 🤔

Reply:
> so it's possible to send a cacheControlMaxAge option from the client by manually creating a token

Yes, by using a read write token on the client (not recommended but 🤷 ).
Yes, by creating a client token and using `put()`. In this case, this is where we could (maybe) add runtime checks and (for sure) server-side checks on what is received.

Reply:
then I don't see the advantages of adding a `clientPut`, the additional typescript interface `{handleClientUploadUrl} | {cacheControlMaxAge,...}` would be enough...are there other reasons to create a `clientPut`?

Reply:
Yes, I think it clarifies the intent and behavior of the put method when you want to use the client upload helpers versus having a method that changes its behavior based on its arguments.